test/snippet/io/structure_file/structure_file_input.cpp

```diff
@@ -0,0 +1,193 @@
+#include <iostream>
+
+#include <seqan3/io/structure_file/input.hpp>
+#include <seqan3/range/view/get.hpp>
+#include <seqan3/std/view/filter.hpp>
+
+using namespace seqan3;
+
+// For col_read snippet
+struct data_storage_t
```

Comment:
again always put stuff into the snippet so it is easier for the user to see how stuff works